assets/properties-panel.css

```diff
@@ -1,1435 +1,4 @@
-  width: 100%;
-}
-
-.bio-properties-panel-feel-popup .bio-properties-panel-feel-editor-container {
-  display: flex;
-  min-width: 100%;
-}
-
-.bio-properties-panel-feel-popup .bio-properties-panel-feelers-editor-container {
-  width: 100%;
-  display: flex;
-}
-
-.bio-properties-panel-feel-popup .bio-properties-panel-feelers-editor-container .bio-properties-panel-feelers-editor {
-  display: flex;
-}
-
-.bio-properties-panel-feel-popup .bio-properties-panel-feelers-editor-container .bio-properties-panel-feelers-editor .cm-editor {
-  width: 100%;
-}
-
-.bio-properties-panel-feel-popup .bio-properties-panel-input {
-  width: 100%;
-  resize: none;
-  padding: 0;
-  margin-left: -12px;
-  overflow: hidden;
-  overflow-y: auto
-}
-
-.bio-properties-panel-feel-popup .bio-properties-panel-open-feel-popup {
-  display: none !important;
-}
-
-.bio-properties-panel-feelers-editor-container .bio-properties-panel-open-feel-popup,
-.bio-properties-panel-feel-container .bio-properties-panel-open-feel-popup {
-  position: absolute;
-  display: none;
-  right: 0;
-  bottom: -1px;
-  background: none;
-  border: none;
-  color: var(--feel-open-popup-color);
-  cursor: pointer;
-}
-
-.bio-properties-panel-open-feel-popup svg {
-  background: var(--feel-open-popup-background-color);
-}
-
-.bio-properties-panel-feelers-editor-container:hover .bio-properties-panel-open-feel-popup,
-.bio-properties-panel-feel-container:hover .bio-properties-panel-open-feel-popup,
-.bio-properties-panel-feelers-editor-container:focus-within .bio-properties-panel-open-feel-popup,
-.bio-properties-panel-feel-container:focus-within .bio-properties-panel-open-feel-popup {
-  display: block;
-}
-
-.bio-properties-panel-feelers-editor-container .bio-properties-panel-open-feel-popup {
-  bottom: 0;
-}
-
-.bio-properties-panel-feelers-editor-container .bio-properties-panel-open-feel-popup:hover,
-.bio-properties-panel-feel-container .bio-properties-panel-open-feel-popup:hover {
-  color: var(--feel-open-popup-hover-color);
-}
-
-.bio-properties-panel-feel-popup .bio-properties-panel-popup__footer .bio-properties-panel-feel-popup__close-btn {
-  background: var(--feel-popup-close-background-color);
-  width: 66px;
-  font-weight: 400;
-  font-size: 14px;
-  font-family: inherit;
-  color: white;
-  border: none;
-  height: 32px;
-  cursor: pointer;
-}
-
-.bio-properties-panel-feel-popup .bio-properties-panel-feel-editor-container .cm-editor {
-  width: 100%;
-  height: 100%;
-}
-
-.bio-properties-panel-feel-popup .bio-properties-panel-feel-editor-container .cm-editor .cm-content {
-  padding-left: 4px;
-}
-
-.bio-properties-panel-feel-popup .cm-gutters {
-  background-color: var(--feel-popup-gutters-background-color);
-  border: none;
-  padding: 0;
-}
-
-.bio-properties-panel-feel-popup .cm-gutter {
-  min-width: 32px;
-}
-
-.bio-properties-panel-feel-popup .cm-gutters .cm-lineNumbers .cm-gutterElement {
-  text-align: center;
-}
+@import '../dist/assets/properties-panel.css'
```

Comment:
We keep the `assets` file with a link for backwards compatibility

Reply:
Do we have to? I guess we could simply drop it, as this was not supposed to be consumed in the past (was consumed via `*-properties-panel` until now).

Reply:
:+1: